### DISCUSSION TITLE
新加的gpt4-0125效果很好，沉浸搜索什么时候支持？

### DISCUSSION THREAD

**Discussion by scientificamer** [immersive-translate/immersive-translate#1375] [Ideas]
### 新功能 | New Feature

1.新加的gpt4-0125效果很好，沉浸搜索什么时候支持
2.
3.

### 为什么您想要添加这些功能 | Why you want to add these features

1.效果很好
2.
3.

**Comment by theowenyoung** [comment]
可以在设置中选择自定义模型名称即可。

**Comment by scientificamer** [comment]
可拓浏览器对新版支持是不是有点问题？签名问题？


获取Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
